widget/list.go

```diff
@@ -93,6 +93,36 @@ func (l *List) MinSize() fyne.Size {
 	return l.BaseWidget.MinSize()
 }
 
+func (l *List) refreshBase() (*listLayout, map[int]*listItem, fyne.Focusable) {
+	if l.scroller != nil {
+		l.BaseWidget.Refresh()
+		lo := l.scroller.Content.(*fyne.Container).Layout.(*listLayout)
+		visible := lo.visible
+		canvas := fyne.CurrentApp().Driver().CanvasForObject(lo.list)
+		var focused fyne.Focusable
+		if canvas != nil {
+			focused = canvas.Focused()
+		}
+		return lo, visible, focused
+	}
+	return nil, nil, nil
+}
+
+func (l *List) Refresh() {
+	lo, visible, focused := l.refreshBase()
+	for id, item := range visible {
+		item.updated = false
+		lo.setupListItem(item, id, focused == item)
+	}
+}
+
+func (l *List) RefreshItem(id ListItemID) {
```

Comment:
I'm not sure what access this refresh should have.

Reply:
There is an issue (https://github.com/fyne-io/fyne/issues/3826k) open to track adding support for refreshing individual items. If you want to do this for List, then I'm fine with that

Reply:
Nice! I'll leave this as is then.